src/ReachSets/ContinuousPost/BFFPS19/BFFPS19.jl

```diff
@@ -0,0 +1,291 @@
+
+include("reach.jl")
+include("reach_blocks.jl")
+
+init(𝒫::BFFPS19, 𝑆::AbstractSystem, 𝑂::Options) = init!(𝒫, 𝑆, copy(𝑂))
+
+function init!(𝒫::BFFPS19, 𝑆::AbstractSystem, 𝑂::Options)
+    # state dimension for (purely continuous or purely discrete systems)
+    𝑂copy = copy(𝑂)
+    𝑂copy[:n] = statedim(𝑆)
+
+    # solver-specific options (adds default values for unspecified options)
+    𝑂validated = validate_solver_options_and_add_default_values!(𝑂copy)
+
+    # :vars option; default: all variables
+    if haskey_specified(𝒫.options, :vars)
+        𝑂validated[:vars] = 𝒫.options[:vars]
+    else
+        𝑂validated[:vars] = 1:𝑂validated[:n]
+    end
+
+    # :partition option: use 1D blocks
+    if haskey_specified(𝒫.options, :partition)
+        𝑂validated[:partition] = 𝒫.options[:partition]
+    else
+        𝑂validated[:partition] = [[i] for i in 1:𝑂validated[:n]]
+    end
+
+    opD = 𝒫.options[:opD]
+    if opD isa DecomposedDiscretePost
+        HS = 𝒫.options[:HS]
+        constrained_dims = constrained_dimensions(HS)
+        out_vars = opD.options[:out_vars]
+        loc_id = 𝒫.options[:loc_id]
+        temp_vars = unique([out_vars; constrained_dims[loc_id]])
+        opD.options[:temp_vars] = temp_vars
+        guards_constraints = [guard(HS, trans) for trans in out_transitions(HS, loc_id)]
+        𝑂validated[:vars] = temp_vars
+        𝑂validated[:guards_proj] = [project(c, temp_vars) for c in guards_constraints]
+        𝑂validated[:blocks] = compute_blocks(𝑂validated[:vars], 𝑂validated[:partition])
+
+        info("- Total")
+    else
+        throw(MethodError("This continuous post operator works only with DecomposedDiscretePost"))
+    end
+
+    # :block_options_init & :block_options_iter options:
+    # set default according to :partition
+    if !haskey_specified(𝒫.options, :block_options_init)
+        𝑂validated[:block_options_init] =
+            compute_default_block_options(𝑂validated[:partition])
+    end
+    if !haskey_specified(𝒫.options, :block_options_iter)
+        𝑂validated[:block_options_iter] =
+            compute_default_block_options(𝑂validated[:partition])
+    end
+
+    # Input -> Output variable mapping
+    𝑂validated[:inout_map] = inout_map_reach(𝑂validated[:partition], 𝑂validated[:blocks], 𝑂validated[:n])
+
+    if 𝑂validated[:project_reachset]
+        𝑂validated[:output_function] = nothing
+    else
+        𝑂validated[:output_function] = 𝑂validated[:projection_matrix]
+    end
+
+    return 𝑂validated
+end
+
+"""
+    post(𝒫::BFFPS19, 𝑆::AbstractSystem, 𝑂::Options)
+
+Calculate the reachable states of the given initial value problem using `BFFPS19`.
+
+### Input
+
+- `𝒫` -- post operator of type `BFFPS19`
+- `𝑆` -- sytem, initial value problem for a continuous ODE
+- `𝑂` -- algorithm-specific options
+"""
+function post(𝒫::BFFPS19, 𝑆::AbstractSystem, 𝑂_input::Options)
+    𝑂 = TwoLayerOptions(merge(𝑂_input, 𝒫.options.specified), 𝒫.options.defaults)
+
+    if 𝑂[:mode] == "reach"
```

Comment:
Since you only support `"reach"`, you should do
```julia
@assert 𝑂[:mode] == "reach" "the mode $(𝑂[:mode]) is not supported"
```
and only keep the `if` body.